README.Rmd

```diff
@@ -19,46 +19,80 @@ knitr::opts_chunk$set(
+### Installation
+
+You can install the development version of ccaa.inspectEHR from [GitHub](https://github.com/DocEd/d.inspectEHR) with:
 
 ``` r
 remotes::install_github("DocEd/d.inspectEHR")
 ```
 
-## Running the Report
+To run the reports as an end user, install from github first (will only work if you have access to the repo). If you don't have a personal access token, generate it using the instructions here. https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
 
-The report can be run by loading the package and running the following code.
+Then run the function below using the token generated.
 
-```r
-library(d.inspectEHR)
-render_report(output = "DECOVID_report.html")
+``` r
+devtools::install_github("NICST-PROTECT/ccaa.inspectEHR", auth_token = "your_PAT")
 ```
 
-This will prompt a shiny interface to launch for you to enter your database connection details.
-You can also supply these directly:
+### Running the Report
+
+The report can be run by running the following code.
+
+- The date range that you need to generate the report should be give with *start_date* and *end_date*. 
+- The acceptable tolerance for measurement checks can be given using the *measurement_tolerance*
 
 ```r
 library(d.inspectEHR)
 
-render_report(
-  output = "DECOVID_report.html",
-  prams = list(
+render_summary_report(
+  output = "Summary_report.html",
+  params = list(
+    driver = "PostgreSQL",
+    host = "localhost",
+    port = 5432,
+    dbname = "cca_omop",
+    schema = "cca_omop",
+    user = #####,
+    password = #####,
+    local_hospital = "ccaa",
+    start_date = '01-01-2020',
+    end_date = '31-10-2022'
+  )
+)
+
+render_quality_report(
+  output = "Data_Quality_report.html",
+  params = list(
     driver = "PostgreSQL",
     host = "localhost",
     port = 5432,
-    dbname = "omop",
-    schema = "public",
-    user = "w.flemming",
-    password = "chromatin",
-    local_hospital = "St. Elsewhere"
-    )
+    dbname = "cca_omop",
+    schema = "cca_omop",
+    user = ######,
+    password = ######,
+    local_hospital = "ccaa",
+    start_date = '01-01-2020',
+    end_date = '31-10-2022',
+    measurement_tolerance = '0.01'
+    
   )
+)
 ```
 
-Grab a coffee and enjoy.
+## Run as developer
+
+If you intend to edit code, clone the repo, double click the .Rproj file to open, and use devtools::load_all() to make the functions available in the environment. Follow the instructions above to actually knit the reports. General information on package development here. https://r-pkgs.org
+
+### Rules for contributing
+
+- Create a new branch and send Aasiyah a pull request review.
+- Also share sample generated PDFs.
+- Never add generated PDFs to this repo, because it increases the package size.
+- Use roxygen documentation for any functions you create. If it's not meant to be accessible to users, add the @noRd tag.
```

Comment:
Why haven't my edits to this file appeared?
Were they on the wrong branch?

Reply:
Oh, it's a different file!
This is README.Rmd, not README.md.
Please feel free to delete this file.

Reply:
Sorry, it looks like my mistake in editing the md file instead of the Rmd one. Do you know why it's generated this way?

Reply:
I removed the Rmd one.